xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-preferences/xwiki-platform-notifications-preferences-api/pom.xml

```diff
@@ -65,4 +65,44 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-maven-plugin</artifactId>
+        <configuration>
+          <analysisConfiguration>
+            <revapi.differences>
+              <differences>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.method.addedToInterface</code>
+                  <new>method org.xwiki.notifications.preferences.email.NotificationEmailDiffType org.xwiki.notifications.preferences.email.NotificationEmailUserPreferenceManager::getDiffType(org.xwiki.model.reference.DocumentReference)</new>
+                  <justification>This change is necessary to implement XWIKI-19756</justification>
+                </item>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.method.addedToInterface</code>
+                  <new>method org.xwiki.notifications.preferences.NotificationEmailInterval org.xwiki.notifications.preferences.email.NotificationEmailUserPreferenceManager::getInterval()</new>
+                  <justification>This change is necessary to implement XWIKI-19756</justification>
+                </item>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.method.addedToInterface</code>
+                  <new>method org.xwiki.notifications.preferences.NotificationEmailInterval org.xwiki.notifications.preferences.email.NotificationEmailUserPreferenceManager::getInterval(java.lang.String)</new>
+                  <justification>This change is necessary to implement XWIKI-19756</justification>
+                </item>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.method.addedToInterface</code>
+                  <new>method org.xwiki.notifications.preferences.NotificationEmailInterval org.xwiki.notifications.preferences.email.NotificationEmailUserPreferenceManager::getInterval(org.xwiki.model.reference.DocumentReference)</new>
+                  <justification>This change is necessary to implement XWIKI-19756</justification>
+                </item>
+              </differences>
+            </revapi.differences>
+          </analysisConfiguration>
+        </configuration>
```

Comment:
Also note that this not setting a criticality (see https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HRevapiIgnores).

But in any case this needs to be discussed/voted if backward compatibility is broken.